content/blog/2025-02-20-datafusion-45.0.0.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: post
+title: Apache DataFusion 45.0.0 Released
+date: 2025-02-20
```

Comment:
The site still renders a "author" even though there is no author listed 🤔 

<img width="803" alt="Screenshot 2025-02-22 at 6 10 11 AM" src="https://github.com/user-attachments/assets/c5d264a3-d836-477e-8c38-c74d85a99ba7" />

I think it would be good to mark yourself as the author if you are ok with it - this is a great piece of writing and I think you should get the credit for it. It also shows diversity in the writers of DataFusion content

```suggestion
date: 2025-02-20
author: Bruce Ritchie
```

If you would prefer you could also list the author as pmc